core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -264,6 +273,22 @@ private[log] class LogCleanerManager(val logDirs: Seq[File],
     }
   }
 
+  /**
+   *  Check if the cleaning for a partition is paused. The caller is expected to hold lock while making the call.
+   */
+  private def checkCleaningPaused(topicPartition: TopicPartition): Boolean = {
```

Comment:
there is another function "checkCleaningAborted" , I name this function after that.

Reply:
The existing method `checkCleaningAborted()` is a unit function which throw exception if the state is not `LogCleaningAborted`. The method `isCleaningInState(...)` returns boolean to indicate whether the state has the given value. It seems that the method we are adding here is closer to the `isCleaningInState(...)` since this method returns a boolean instead of throwing exception.